frontend/src/services/userService.js

```diff
@@ -1,6 +1,12 @@
 import axios from 'axios'
 const baseUrl = 'http://localhost:8000/api/user'
```

Comment:
Suggest to add a `process.env.BASE_URL` alternative as well so it can be used during deployment. I'm not sure what's the best way or if this is the correct way though.

`const baseUrl = process.env.ENV == "PROD" ? process.env.CLOUD_ENDPOINT + '/api/user' : http://localhost:8000/api/user`

Reply:
Will take note of this, currently there is no env file, will check and handle this in a future pr/commit.